spec/authorization_spec.rb

```diff
@@ -159,6 +168,38 @@
 
       expect(controller.policy(post)).to eq new_policy
     end
+
+    it "uses the Admin namespace for policy calls" do
+      policy = Project::Admin::CommentPolicy.new(user, comment)
+
+      expect(policy.update?).to eq(true)
+    end
+
+    context "when the policy is overriden" do
+      it "returns the correct overriden policy for a Post record" do
+        post = Post.new(user)
+
+        policy = controller.policy(post)
+        expect(policy).to be_instance_of(PostPolicy)
+        expect(policy.user).to eq(user)
+        expect(policy.post).to eq(post)
```

Comment:
This `context` says "when the policy is overridden", but it doesn't look like anything is being overriden? What's the "policy" which is being overridden (do you mean the `policy` method, or the `Policy` class?), and where is it overridden? 

I see there are some new things added in the `spec_helper` but none of those override `policy_class`, `authorize`, `policy`, `Post` or `PostPolicy`, so this spec looks like it just tests that `::PostPolicy` works - nothing to do with any overriding? 

What am I missing? 

If I understand it correctly, the "overriding" which the original issue was talking about defining `policy` or `authorize` in a specific controller - eg. in an admin controller you might want to do something like this:

```ruby
  def authorize(record, query = nil)
    super([:admin, record], query)
  end

  def policy(record)
    super([:admin, record])
  end 
```

...to make sure that calls to `authorize(Post)` and `policy(Post)` in that controller end using the `Admin::PostPolicy` instead of `::PostPolicy`.

I'm actually using this exact override of `authorize` in my current project.